2022-05-01T19:58:48.002Z	debug	1.3.0@a2d6c4b	Initializing API credentials	{"file": "/chainlink/apicredentials"}
2022-05-01T19:58:48.010Z	info	1.3.0@a2d6c4b.RunNode	Starting Chainlink Node 1.3.0 at commit a2d6c4b47d62ac792a215b3a8d3acfe3ae9a2966	{"Version": "1.3.0", "SHA": "a2d6c4b47d62ac792a215b3a8d3acfe3ae9a2966"}
2022-05-01T19:58:48.011Z	error	1.3.0@a2d6c4b	Cannot boot Chainlink: opening db: failed to open db: failed to connect to `host=/tmp user=anniehua database=`: dial error (dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory)	{"err": "Cannot boot Chainlink: opening db: failed to open db: failed to connect to `host=/tmp user=anniehua database=`: dial error (dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory)", "errVerbose": "opening db: failed to open db: failed to connect to `host=/tmp user=anniehua database=`: dial error (dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory)\nCannot boot Chainlink\ngithub.com/smartcontractkit/chainlink/core/cmd.(*Client).RunNode\n\t/chainlink/core/cmd/local_client.go:53\ngithub.com/urfave/cli.HandleAction\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:524\ngithub.com/urfave/cli.Command.Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:173\ngithub.com/urfave/cli.(*App).RunAsSubcommand\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:405\ngithub.com/urfave/cli.Command.startApp\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:372\ngithub.com/urfave/cli.Command.Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:102\ngithub.com/urfave/cli.(*App).Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:277\nmain.Run\n\t/chainlink/core/main.go:25\nmain.main.func1\n\t/chainlink/core/main.go:18\ngithub.com/smartcontractkit/chainlink/core/recovery.ReportPanics\n\t/chainlink/core/recovery/recover.go:17\nmain.main\n\t/chainlink/core/main.go:17\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:255\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1581", "sentryEventID": null}
2022-05-01T20:34:32.156Z	debug	1.3.0@a2d6c4b	Initializing API credentials	{"file": "/chainlink/apicredentials"}
2022-05-01T20:34:32.162Z	info	1.3.0@a2d6c4b.RunNode	Starting Chainlink Node 1.3.0 at commit a2d6c4b47d62ac792a215b3a8d3acfe3ae9a2966	{"Version": "1.3.0", "SHA": "a2d6c4b47d62ac792a215b3a8d3acfe3ae9a2966"}
2022-05-01T20:34:32.165Z	error	1.3.0@a2d6c4b	Cannot boot Chainlink: opening db: failed to open db: failed to connect to `host=/tmp user=anniehua database=`: dial error (dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory)	{"err": "Cannot boot Chainlink: opening db: failed to open db: failed to connect to `host=/tmp user=anniehua database=`: dial error (dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory)", "errVerbose": "opening db: failed to open db: failed to connect to `host=/tmp user=anniehua database=`: dial error (dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory)\nCannot boot Chainlink\ngithub.com/smartcontractkit/chainlink/core/cmd.(*Client).RunNode\n\t/chainlink/core/cmd/local_client.go:53\ngithub.com/urfave/cli.HandleAction\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:524\ngithub.com/urfave/cli.Command.Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:173\ngithub.com/urfave/cli.(*App).RunAsSubcommand\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:405\ngithub.com/urfave/cli.Command.startApp\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:372\ngithub.com/urfave/cli.Command.Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:102\ngithub.com/urfave/cli.(*App).Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:277\nmain.Run\n\t/chainlink/core/main.go:25\nmain.main.func1\n\t/chainlink/core/main.go:18\ngithub.com/smartcontractkit/chainlink/core/recovery.ReportPanics\n\t/chainlink/core/recovery/recover.go:17\nmain.main\n\t/chainlink/core/main.go:17\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:255\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1581", "sentryEventID": null}
2022-05-01T20:35:41.639Z	debug	1.3.0@a2d6c4b	Initializing API credentials	{"file": "/chainlink/apicredentials"}
2022-05-01T20:35:41.645Z	info	1.3.0@a2d6c4b.RunNode	Starting Chainlink Node 1.3.0 at commit a2d6c4b47d62ac792a215b3a8d3acfe3ae9a2966	{"Version": "1.3.0", "SHA": "a2d6c4b47d62ac792a215b3a8d3acfe3ae9a2966"}
2022-05-01T20:35:41.652Z	error	1.3.0@a2d6c4b	Cannot boot Chainlink: opening db: failed to open db: cannot parse `postgresql://dev:xxxxx@127.0.0.1:5432:5432/chainlink?application_name=Chainlink+1.3.0+%7CApp%7Ce1948c86-3d99-4799-929b-e3350622ea45`: failed to parse as URL (failed to split host:port in '127.0.0.1:5432:5432', err: address 127.0.0.1:5432:5432: too many colons in address)	{"err": "Cannot boot Chainlink: opening db: failed to open db: cannot parse `postgresql://dev:xxxxx@127.0.0.1:5432:5432/chainlink?application_name=Chainlink+1.3.0+%7CApp%7Ce1948c86-3d99-4799-929b-e3350622ea45`: failed to parse as URL (failed to split host:port in '127.0.0.1:5432:5432', err: address 127.0.0.1:5432:5432: too many colons in address)", "errVerbose": "opening db: failed to open db: cannot parse `postgresql://dev:xxxxx@127.0.0.1:5432:5432/chainlink?application_name=Chainlink+1.3.0+%7CApp%7Ce1948c86-3d99-4799-929b-e3350622ea45`: failed to parse as URL (failed to split host:port in '127.0.0.1:5432:5432', err: address 127.0.0.1:5432:5432: too many colons in address)\nCannot boot Chainlink\ngithub.com/smartcontractkit/chainlink/core/cmd.(*Client).RunNode\n\t/chainlink/core/cmd/local_client.go:53\ngithub.com/urfave/cli.HandleAction\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:524\ngithub.com/urfave/cli.Command.Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:173\ngithub.com/urfave/cli.(*App).RunAsSubcommand\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:405\ngithub.com/urfave/cli.Command.startApp\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:372\ngithub.com/urfave/cli.Command.Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/command.go:102\ngithub.com/urfave/cli.(*App).Run\n\t/go/pkg/mod/github.com/urfave/cli@v1.22.5/app.go:277\nmain.Run\n\t/chainlink/core/main.go:25\nmain.main.func1\n\t/chainlink/core/main.go:18\ngithub.com/smartcontractkit/chainlink/core/recovery.ReportPanics\n\t/chainlink/core/recovery/recover.go:17\nmain.main\n\t/chainlink/core/main.go:17\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:255\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1581", "sentryEventID": null}
